docs/developer-docs/latest/update-migration-guides/migration-guides/v4/migration-guide-4.0.x-to4.0.6.md

```diff
@@ -41,10 +45,19 @@ module.exports = [
 ];
 ```
 
-### 3. Configure session middleware
 
-The session middleware requires some keys to secure the cookies and encrypt data, for more information on how this works please see [koa-session](https://github.com/koajs/session/blob/master/Readme.md). In order to properly configure the middleware you will need to add these key settings to your `server.js` config file:
+2. Configure the session middleware by adding the key settings to the `server.js` config file (see [koa-session](https://github.com/koajs/session/blob/master/Readme.md) for more information).
+ 
 
+```jsx
+// path: ./config/server.js
```

Comment:
```suggestion
// path: ./config/server.js

```

Really nitpicking here, but we use a blank line between the "path" comment line and the actual code, so it's easier to distinguish between the two :D